pkg/kubectl/cmd/util/helpers.go

```diff
@@ -349,15 +349,24 @@ func GetFlagInt(cmd *cobra.Command, flag string) int {
 func GetFlagInt64(cmd *cobra.Command, flag string) int64 {
 	i, err := cmd.Flags().GetInt64(flag)
 	if err != nil {
-		glog.Fatalf("err accessing flag %s for command %s: %v", flag, cmd.Name(), err)
+		glog.Fatalf("error accessing flag %s for command %s: %v", flag, cmd.Name(), err)
+	}
+	return i
+}
+
+// Assumes the flag has a default value.
+func GetFlagInt32(cmd *cobra.Command, flag string) int32 {
```

Comment:
@AdoHe It is no longer needed, I went ahead and removed it https://github.com/kubernetes/kubernetes/pull/33319/commits/577b5a548a763086ad4ddf60ef8aab15237e4dc4